internal/jobs/launch_instance_azure.go

```diff
@@ -95,6 +91,7 @@ func DoEnsureAzureResourceGroup(ctx context.Context, args *LaunchInstanceAzureTa
 
 	azureClient, err := clients.GetAzureClient(ctx, args.Subscription)
 	if err != nil {
+		logger.Error().Err(err).Msg("cannot create new Azure client")
```

Comment:
```suggestion
		logger.Error().Err(err).Msg("Cannot create new Azure client")
```

Reply:
Lesson learned for my next project - all logging start with lower case :-)